.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
We love this ESlint. I could not check whether you have .eslintignore so you can ignore the eslint for bundle.js. If you have not set up yet, I recommend you to do it :)